client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -155,6 +161,13 @@ class Theme extends Component {
 		return sprintf( __( '%s per year', 'woocommerce-admin' ), decodeEntities( price ) );
 	}
 
+	doesCurrentThemeSupportWooCommerce() {
```

Comment:
Minor and nitpicky, but semantically I like `activeTheme` over `currentTheme` since this has less ambiguous meaning (e.g., current theme inside a live preview demo vs currently active theme on the site).